src/Console/Command/Ci/CiRunCommand.php

```diff
@@ -131,16 +142,24 @@ public function execute(InputInterface $input, OutputInterface $output): int {
       $data['status'] = 'FAIL';
       $job = $this->ciJobFactory->create($options->getJob());
       $data['version'] = $job->getDrupalCoreVersion();
+      $data['allowedToFail'] = $this->env->get("ORCA_IS_ALLOWED_FAILURE") ?? FALSE;
+      $data['branch'] = $this->env->get("ORCA_SUT_BRANCH") ?? FALSE;
       $job->run($options);
 
     }
     catch (OrcaInvalidArgumentException $e) {
       $output->writeln("Error: {$e->getMessage()}");
+      if ($this->env->get("ORCA_IS_ALLOWED_FAILURE")) {
+        return StatusCodeEnum::OK;
+      }
```

Comment:
@TravisCarden : Do you think this would be problematic?

Reply:
No, I think I would let this one return an error code, because it's not a test failure that should be allowable--it means that something broke with the setup that needs to be surfaced and fixed.

Reply:
okay